onnxruntime/core/common/path.cc

```diff
@@ -253,18 +253,16 @@ Path& Path::Append(const Path& other) {
   return *this;
 }
 
-Path& Path::Concat(const PathString& string) {
-  components_.back() += string;
-  return *this;
-}
-
-Path& Path::ConcatIndex(const int index) {
-#ifdef _WIN32
-  auto index_str = std::to_wstring(index);
-#else
-  auto index_str = std::to_string(index);
-#endif
-  components_.back() += index_str;
+Path& Path::Concat(const PathString& value) {
+  auto first_separator = std::find_if(value.begin(), value.end(), IsPreferredPathSeparator);
```

Comment:
probably want to check against k_valid_path_separators instead

Reply:
Ok.